source/_integrations/ibeacon.markdown

```diff
@@ -74,6 +74,7 @@ To get the Estimated distance sensor to work, in most cases, it has to be calibr
 - [Feasycom FSC-BP103B](https://www.feasycom.com/bluetooth-ibeacon-da14531)
 - [Feasycom FSC-BP104D](https://www.feasycom.com/dialog-da14531-bluetooth-low-energy-beacon)
 - [Feasycom FSC-BP108](https://www.feasycom.com/bluetooth-5-1-waterproof-bluetooth-beacon)
+- [Mikrotik TG-BT5-IN](https://mikrotik.com/product/tg_bt5_in) (Additional sensors such as angle, impact, etc are not compatible)
```

Comment:
```suggestion
- [Mikrotik TG-BT5-IN](https://mikrotik.com/product/tg_bt5_in) (Additional sensors such as angle or impact are not compatible)
```
Tiny tweak to comply with [style guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/etc).